pkg/otlp/internal/serializerexporter/exporter.go

```diff
@@ -176,8 +176,8 @@ func (e *exporter) ConsumeMetrics(ctx context.Context, ld pmetric.Metrics) error
 	}
 
 	consumer.addTelemetryMetric(e.hostname)
-	if rmt.HasRuntimeMetrics {
-		consumer.addRuntimeTelemetryMetric(e.hostname, rmt.LanguageTags)
+	if len(rmt.Languages) != 0 {
```

Comment:
You don't need this if condition, `addRuntimeTelemetryMetric` won't do anything if `rmt.Languages` is empty.